src/routes/contributors/+layout.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import Header from '$lib/components/organisms/Header.svelte';
+	import Footer from '$lib/components/organisms/Footer.svelte';
+</script>
+
+<Header showBackground />
+
+<div class="contributor-biography">
+	<h1>Contributors</h1>
+	<slot />
+</div>
+
+<Footer />
+
+<style>
+	.contributor-biography {
+		display: flex;
+		flex-direction: column;
+		justify-content: space-between;
+		text-align: center;
+		max-width: 700px;
+		margin: 0 auto;
+		margin-top: 2rem;
+		line-height: 1.5;
+	}
```

Comment:
Hi @grmbyrn why do you need this also here? It's also defined [inside the component](https://github.com/torrust/torrust-website/pull/108/files#diff-26602b02881602b43ba9d57dded530229319993290ded5addaf9ef90fa5b653eR49-R59).

Reply:
Hi @josecelano. I added the contributor link to each article and removed duplicated css.